package.json

```diff
@@ -25,21 +25,21 @@
     "eslint"
   ],
   "peerDependencies": {
-    "eslint-plugin-mocha": "^1.0.0",
-    "eslint-plugin-react": "^3.3.2",
-    "eslint-plugin-json": "^1.1.0"
+    "eslint-plugin-mocha": "^2.0.0",
+    "eslint-plugin-react": "^3.16.1",
+    "eslint-plugin-json": "^1.2.0"
   },
   "dependencies": {
     "babel-eslint": "^4.1.3",
-    "escope": "3.3.0",
-    "eslint": "^1.8.0",
+    "escope": "^3.4.0",
```

Comment:
If you update babel-eslint to 4.1.8 you can remove this. https://github.com/babel/babel-eslint/issues/243